src/react/actions/__tests__/s3object.test.jsx

```diff
@@ -0,0 +1,118 @@
+import * as actions from '../s3object';
+import * as dispatchAction from './utils/dispatchActionsList';
+import {
+    BUCKET_NAME, COMMON_PREFIX,
+    FILE, FOLDER_NAME,
+    S3_OBJECT, errorS3State,
+    initState,
+    testActionFunction, testDispatchFunction,
+} from './utils/testUtil';
+
+const createFolderNetworkAction = dispatchAction.NETWORK_START_ACTION('Creating folder');
+const listObjectsNetworkAction = dispatchAction.NETWORK_START_ACTION('Listing objects');
+const uploadObjectsNetworkAction = dispatchAction.NETWORK_START_ACTION('Uploading object(s)');
+
+describe('s3object actions', () => {
+    const syncTests = [
+        {
+            it: 'should return LIST_OBJECTS_SUCCESS action',
+            fn: actions.listObjectsSuccess([S3_OBJECT], [COMMON_PREFIX], ''),
```

Comment:
Could we also try it a non-empty prefix?

Reply:
done